index.md

```diff
@@ -195,96 +129,113 @@ either the result of a digital migration or information that enables the creatio
 of an emulation environment to render a representation. The result of this
 operation is the creation of a new version of the AIP.
```

Comment:
Is this always true, a new version of an AIP could contain fewer reps in the case of deletion? It's a bit of a technicality and maybe best left unmentioned

Reply:
Reformulated.